src/v1.js

```diff
@@ -628,26 +628,49 @@ api.declare({
     "",
     "The caller's scopes must satisfy all of the new scopes being added, but",
     "need not satisfy all of the client's existing scopes.",
+    "",
+    "As a special case an priviledged caller with the scope:",
+    "`auth:manage-scope:<scope>` may, add or remove `<scope>` from any role.",
+    "Notice, that `auth:manage-scope:<scope>` doesn't allow for modifications",
+    "of the description or other scopes even if the caller has such scopes.",
+    "",
+    "The `auth:manage-scope:<scope>` is given to someone who is trusted to",
+    "manage `<scope>`, thus, has the authority to grant the scope to anyone.",
+    "Naturally, this authority comes with the responsibility of having to",
+    "remove the scope from roles that don't need it.",
   ].join('\n')
 }, async function(req, res) {
   let roleId    = req.params.roleId;
   let input     = req.body;
 
-  // Check scopes
-  if (!req.satisfies({roleId})) {
-    return;
-  }
-
   // Load role
   let role = await this.Role.load({roleId}, true);
   if (!role) {
     return res.status(404).json({message: "role not found!"});
   }
 
-  // Check that requester has all the scopes added
-  let added = _.without.apply(_, [input.scopes].concat(role.scopes));
-  if (!req.satisfies([added])) {
-    return;
+  // Find scopes added and removed
```

Comment:
All of this arithmetic will need to be done within the modify, to avoid race conditions.